src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryDate.cs

```diff
@@ -2,12 +2,15 @@
 // Licensed under the MIT license.
 
 using System;
+using System.Collections.Generic;
```

Comment:
Needed? Try running VS's Remove and Sort usings to clean it up

Reply:
Turns out that VS's Remove and Sort removed things, but it no longer compiled.  :)  I had to put some back.